plugins/scaffolder-backend/sample-templates/v1beta2-demo/template/catalog-info.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: {{cookiecutter.name | jsonify}}
```

Comment:
ok not to quote this? feels dangerous :)

Reply:
jsonify quotes it already

Reply:
Yeah but I mean that the YAML parser (far earlier than that) might want to interpret the curly brace as the start of an object. It's probably ok if there is no space after the first curly but I'd be tempted to be better safe than sorry and quote it `'{{}}'` like in the other places.

Reply:
It is cookie-cut before it's is YAMLized

Reply:
Ah that's true...